straxen/contexts.py

```diff
@@ -62,7 +62,10 @@ def xenonnt_online(output_folder='./strax_data',
         readonly=not we_are_the_daq,
         runid_field='number',
         new_data_path=output_folder,
-        rucio_path='/dali/lgrandi/rucio/'), ] if _database_init else []
+        rucio_path='/dali/lgrandi/rucio/'),
+        straxen.OnlineMonitor(readonly=not we_are_the_daq,
```

Comment:
@darrylmasson we could move this line under an ``if we_are_the_daq`` to prevent a multitude of queries. Any expert strax user could later do 

```python
st.storage += [OnlineMonitor(readonly=True, take_only=('pulse_counts', 'pulse_counts_he')]
```